docs/README.md

```diff
@@ -1,30 +1,23 @@
-## NEURON Development
+## NEURON Documentation
 
-### Versioning and Release
-Please refer to [NEURON Versioning and Release Guidelines](./scm/guidelines/SCMGuidelines.md).
-
-### Source and Release Management Guide
-Please refer to [NEURON Source and Release Management Guide](./scm/guide/SCMGuide.md).
-
-### Developer documentation 
-
-#### Online
+### Online
 Latest NEURON documentation is available on:
+*  ReadTheDocs @ [https://nrn.readthedocs.io/en/latest/](https://nrn.readthedocs.io/en/latest/).
 * GithubPages @ [https://neuronsimulator.github.io/nrn/](https://neuronsimulator.github.io/nrn/).
-* ReadTheDocs @ [https://nrn.readthedocs.io/en/latest/](https://nrn.readthedocs.io/en/latest/).
 
 Contents:
 * documentation on building NEURON
 * user documentation (HOC, Python, tutorials, rxd)
 * developer documentation (SCM, technical topics, Doxygen)
 
-#### Local build
+### Local build
 
-It is recommended using a `virtualenv`, for example:
+#### Setup
+It is recommended to use a Python virtual environment, for example:
```

Comment:
What circumstances make a virtual environment superior?

Reply:
Usual benefits of an isolated environment apply. We do rely on a handful of packages (https://github.com/neuronsimulator/nrn/blob/master/docs/docs_requirements.txt)